mutatex/utils.py

```diff
@@ -191,14 +191,13 @@ def parse_poslist_file(fname):
     except IOError:
         log.error("Couldn't open position list file %s" % fname)
         raise IOError
-
     for line in fh:
         if matcher.fullmatch(line.strip()) is None:
             log.error("the position list file is not in the right format")
             log.error("format error at %s" % line.strip())
             raise TypeError
-
         tmp = line.strip().split("_")
+        residue=tuple(tmp)
```

Comment:
I don't think there are cases here in which you would need a list more of a tuple - so you can just

```
residue = tuple(line.strip().split("_"))
```
and then use `residue` instead of `tmp` in the following lines

Reply:
done